1.10/networking/load-balancing-vips/index.md

```diff
@@ -1,134 +1,51 @@
 ---
-post_title: Load Balancing and VIPs
-nav_title: Load Balancing and VIPs
+post_title: Load Balancing and Virtual IPs (VIPs)
 menu_order: 00
 ---
 
-DC/OS comes with an east-west load balancer that's meant to be used to enable multi-tier microservices architectures. It acts as a TCP Layer 4 load balancer, and it's tightly integrated with the kernel. 
+DC/OS provides an east-west load balancer (minuteman) that enables multi-tier microservices architectures. It acts as a TCP layer 4 load balancer, and leverages load-balancing features within the Linux kernel to achieve near line-rate throughputs and latency. The features include:
 
-## Usage
-You can use the layer 4 load balancer by assigning a [VIP from the DC/OS web interface](/docs/1.10/networking/load-balancing-vips/virtual-ip-addresses/). Alternatively, if you're using something other than Marathon, you can create a label on the [port](https://github.com/apache/mesos/blob/b18f5bf48fda12bce9c2ac8e762a08f537ffb41d/include/mesos/mesos.proto#L1813) protocol buffer while launching a task on Mesos. This label's key must be in the format `VIP_$IDX`, where `$IDX` is replaced by a number, starting from 0. Once you create a task, or a set of tasks with a VIP, they will automatically become available to all nodes in the cluster, including the masters.
+- Included with DC/OS by default 
+- Distributed load balancing of applications
+- Facilitates east-west communication within the cluster
+- User specifies an FQDN address to the DC/OS service
+- Respects health checks
+- Requires maintenance of virtual IP to service mappings
+- Automatically allocates virtual IPs to service FQDN
 
-### Details
-When you launch a set of tasks with these labels, DC/OS distributes them to all of the nodes in the cluster. All of the nodes in the cluster act as decision makers in the load balancing process. A process runs on all the agents that the kernel consults when packets are recognized with this destination address. This process keeps track of availability and reachability of these tasks to attempt to send requests to the right backends.
+You can use the layer 4 load balancer by assigning a [VIP](/docs/1.10/networking/load-balancing-vips/virtual-ip-addresses/) in your app definition. After you create a task, or a set of tasks with a VIP, they will automatically become available to all nodes in the cluster, including the masters.
 
-### Recommendations
+When you launch a set of tasks DC/OS distributes them to all of the nodes in the cluster. All of the nodes in the cluster act as decision makers in the load balancing process. A process runs on all the agents that the kernel consults when packets are recognized with this destination address. This process keeps track of availability and reachability of these tasks to attempt to send requests to the right backends.
```

Comment:
how does this sound: `When you launch a set of tasks, DC/OS distributes them to a set of nodes in the cluster. The minuteman instance running on each of the cluster agents coordinates the load balancing decisions. The Mesos kernel consults the minuteman instance on each agent when packets are recognized with this destination address. Minuteman tracks the availability and reachability of these tasks and attempts to send requests to the right backends.`